charmcraft.yaml

```diff
@@ -18,6 +18,9 @@ bases:
 # This below is needed for charmhelpers 0.20.24 to build
 parts:
   charm:
-    charm-python-packages: [setuptools < 58]
+    charm-python-packages:
+      - setuptools < 58
+    charm-binary-python-packages:
+      - substrate-interface
```

Comment:
Not possible to build from source so had to put it here instead of in the requirements.txt file.